changelogs/fragments/454-sns_topic_fix_sms_endpoint_canonicalization.yaml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - sns_topic - Add `+` to allowable characters in SMS endpoints. `AWS SNS expects phone numbers in and canonicalizes to E.164 format <https://docs.aws.amazon.com/sns/latest/dg/sms_publish-to-phone.html>`_. (https://github.com/ansible-collections/community.aws/pull/454)
```

```suggestion
  - sns_topic - Add ``+`` to allowable characters in SMS endpoints (https://github.com/ansible-collections/community.aws/pull/454).
```

- double ticks are required rather than single ticks
- The PR contains the additional information if folks are interested in the "Why"